src/ResourceManager/EventHub/Commands.EventHub/Models/EventHubsAttributes.cs

```diff
@@ -61,7 +60,7 @@ public EventHubAttributes(EventHubResource ehResource)
         /// <summary>
         /// Current number of shards on the Event Hub.
         /// </summary>
-        public IList<int?> PartitionIds { get; set; }
+        public IList<string> PartitionIds { get; set; }
```

Comment:
Dores the service really accept more than an int? Are you accepting other types of input, or are you just looseining the modeling of this property?